docs/naps/7-key-binding-dispatch.md

```diff
@@ -169,9 +178,25 @@ A. The provided key sequence is a single modifier that is a modifier in another
 
 B. The provided key sequence is a base key or key combination that is the first part of another key binding's key chord. For example, a key combination of `ctrl+l` would conflict with the key chord of `ctrl+l p`.
 
-In case (A), the corresponding command will not be triggered immediately, but will be delayed by user-defined miliseconds (e.g. 200ms), after which the press logic for the command will execute. If another key binding is triggered, this action will be canceled. If the base key is released early, the press logic will execute immediately and the delayed action will be canceled, along with the release logic being executed immediately afterwards.
+There are three potential strategies to deal with this:
+
+* Delay - wait for potential subsequent key presses before executing the command.
+    - this is used for case (A) type conflicts
+    - delay defaults to 200ms but is user configurable
+    - after the delay, the press logic for the command will execute
+    - if another key binding is triggered during delay, this action will be canceled
+    - if the base key `ctrl` is released early, the press logic will execute
+      immediately and the delay will cease, and the release logic will be executed immediately afterwards
+* Execute longest key sequence only - for indirectly conflicting key sequences
+  we only ever execute the longer key sequence.
+    - this is used for case (B) type conflicts
+    - the command for `ctrl+l` will never be triggered so long as it indirectly
+      conflicts with another key binding
+    - multi-part key bindings will always take priority over single-part key bindings
+* On release only - only allow the key to be bindable to 'on-release'
+    - Decided against as this would not allow both 'on-press' and 'on-release' actions.
 
-In case (B), the corresponding command will never be triggered so long as it indirectly conflicts with another key binding. In this sense, multi-part key bindings will always take priority over single-part key bindings.
+In case (A), the corresponding command will not be triggered immediately, but will be delayed by user-defined miliseconds (e.g. 200ms), after which the press logic for the command will execute. If another key binding is triggered, this action will be canceled. If the base key is released early, the press logic will execute immediately and the delayed action will be canceled, along with the release logic being executed immediately afterwards.
```

Comment:
Is it just me or is this sentence identical to the `Delay` section above? If not, perhaps it would be good to have a comment on how this differs